### DISCUSSION TITLE
Getting remote address on POST

### DISCUSSION THREAD

**Discussion by epsilon-0** [uNetworking/uWebSockets#1100] [Q&A] [answered]
So far I have only seen how to get remote address if using a websocket and using the asyncsocket getRemoteAddressAsText().

I don't know if it is possible to get the remote address from the POST.

I tried reading the code but so far don't see it.

Thanks,
Aisha

**Comment by ghost** [comment, marked as answer]
HttpResponse is an AsyncSocket so you have it there

**Comment by epsilon-0** [comment]
Ah, it was in the response!!!
Thanks a lot, :smile_cat:.
